src/redteam/strategies/simpleImage.ts

```diff
@@ -0,0 +1,106 @@
+import { SingleBar, Presets } from 'cli-progress';
+import sharp from 'sharp';
+import logger from '../../logger';
+import type { TestCase } from '../../types';
+import invariant from '../../util/invariant';
+
+// Helper function to escape XML special characters for SVG
+function escapeXml(unsafe: string): string {
+  return unsafe
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&apos;');
+}
+
+/**
+ * Converts text to an image and then to base64 encoded string
+ * using the sharp library which has better cross-platform support than canvas
+ */
+export async function textToImage(text: string): Promise<string> {
+  try {
+    // Create a simple image with the text on a white background
+    // We're using SVG as an intermediate format as it's easy to generate without canvas
+    const svgImage = `
+      <svg width="800" height="400" xmlns="http://www.w3.org/2000/svg">
+        <rect width="100%" height="100%" fill="white"/>
+        <text x="50" y="50" font-family="Arial" font-size="16" fill="black">${escapeXml(text)}</text>
+      </svg>
+    `;
+
+    // Convert SVG to PNG using sharp
+    const pngBuffer = await sharp(Buffer.from(svgImage)).png().toBuffer();
```

Comment:
instead of requiring sharp we could put this into our backend or even use a service like quickchart